ogb/lsc/mag240m.py

```diff
@@ -108,14 +128,14 @@ def all_paper_year(self) -> np.ndarray:
         path = osp.join(self.dir, 'processed', 'paper', 'node_year.npy')
         return np.load(path)
 
-    def edge_index(self, id1: str, id2: str,
-                   id3: Optional[str] = None) -> np.ndarray:
-        src = id1
-        rel, dst = (id3, id2) if id3 is None else (id2, id3)
-        rel = self.__rels__[(src, dst)] if rel is None else rel
-        name = f'{src}___{rel}___{dst}'
-        path = osp.join(self.dir, 'processed', name, 'edge_index.npy')
-        return np.load(path)
+    # def edge_index(self, id1: str, id2: str,
```

Comment:
Uncomment back in?

Reply:
This function `edge_index` is no need any more. The edge_index info can be found in `data[(('author', 'writes', 'paper'))].edge_index`, `data[('author', 'affiliated_with', 'institution')].edge_index` and `data[('paper', 'cites', 'paper')].edge_index`, right?